### DISCUSSION TITLE
Quarto - Insert size visualization

### DISCUSSION THREAD

**Discussion by marchoeppner** [bio-raum/FooDMe2#74] [General]
Hi,

just a quick thing regarding how the insert size plots is drawn currently and maybe a "concern" related to that. 

Basically, the plot now uses the "hover" feature, and is configured to show the count for all samples at any given position. 

I can see situations where that is not a desirable behaviour. Specifically, the FooDMe1 test data has 1-2 samples with a small peak leading the expected main peak:

![image](https://github.com/user-attachments/assets/47b4df84-e442-4bd3-b872-06a85f85552f)

But since the hover template will list all samples (only twenty, really, the rest gets truncated), one cannot use that plot to identify samples that have an unexpected (secondary) peak unless it so happens to be among the twenty samples shown.

I think the original behaviour was that the hover function would name the sample a data point belongs too. Obviously, that also only works when there aren't any samples overlapping. 

So either way is a little sub-optiomal. Cannot think if a better solution right now...just thought I'd point it out tho. 

/M

**Comment by gregdenay** [comment]
I changed the tooltips mode to `closest`
